VisualRust.Build/Message/Json/RustcMessageJsonParser.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace VisualRust.Build.Message.Json
+{
+    public static class RustcMessageJsonParser
+    {
+        private static readonly JsonSerializer serializer = new JsonSerializer();
+
+        public static IEnumerable<RustcMessageJson> Parse(String output)
+        {
+            var reader = new JsonTextReader(new StringReader(output)) {SupportMultipleContent = true};
```

Comment:
I assume that the JSON deserialization keeps working if the format is changed (hopefully only backwards compatibly, i.e. by adding new fields), right?